feat(curriculum): forum leaderboard project (#50241)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/646494e11d0cab03caee204c.md

---
id: 646494e11d0cab03caee204c
title: Step 81
challengeType: 0
dashedName: step-81
---

# --description--

> When anyone clicks on the post title, they should be able to read the entire post on a different page, for that, replace the `p` tag with an HTML anchor `a` tag

Reviewer: ```suggestion
When a user clicks on the post title, they should be able to read the entire post on the freeCodeCamp forum.

Replace the `p` tags with `a` tags.
```

# --hints--

> You should replace the `p` tag with an `a` tag, please set the class name to `post-title`

Reviewer: ```suggestion
You should replace the `p` tags with `a` tags.
```
